mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -5191,6 +5231,18 @@ func expandEnableK8sBetaApis(configured interface{}, enabledAPIs []string) *cont
 	return result
 }
 
+func expandClusterNetworkPerformanceConfig(configured interface{}) *container.ClusterNetworkPerformanceConfig {
+	l := configured.([]interface{})
+	if len(l) == 0 || l[0] == nil {
+		return nil
+	}
+
+	config := l[0].(map[string]interface{})
+	return &container.ClusterNetworkPerformanceConfig{
+		Tier: config["totalEgressBandwidthTier"].(string),
```

Comment:
The problem might be that this field needs to be called `TotalEgressBandwidthTier` - [the docs don't show a field called Tier for ClusterNetworkPerformanceConfig](https://pkg.go.dev/google.golang.org/api/container/v1#ClusterNetworkPerformanceConfig).

I'm not sure why the problem appeared to be specific to the beta version of the provider when I investigated the [crash mentioned here](https://github.com/GoogleCloudPlatform/magic-modules/pull/8841#issuecomment-1703090373), maybe it just happened to be the beta version that encountered the problem first